modules/repository-url/src/main/java/org/opensearch/repositories/url/URLRepository.java

```diff
@@ -85,10 +86,10 @@ public class URLRepository extends BlobStoreRepository {
         Property.NodeScope
     );
 
-    public static final Setting<URL> URL_SETTING = new Setting<>("url", "http:", URLRepository::parseURL, Property.NodeScope);
+    public static final Setting<URL> URL_SETTING = new Setting<>("url", "http:/", URLRepository::parseURL, Property.NodeScope);
```

Comment:
The previous default value of `http:` is not parseable as a URI but `http:/` is, hence the change here.

Reply:
Just to note, with "http:/" , the `/` will be interpreted as a path (which with `http:` used to be empty)

Reply:
Not sure it is better. but may be `URI.create("http://?")` or `URI.create("http://#")` are a bit better replacements, at least the path stays empty,

Reply:
Sure, I'll change it to `http://?`